### PR TITLE
Added music controller and ability to get top tracks/playlists

### PR DESCRIPTION
**Pull request by AlexeyZatsepin**

### CODE REVIEW

**Comment by uaArsen** [comment]
Should we write some tests for this code? I can write some if you want. Also I suggest you to log some thing.

File: server/src/main/java/org/room/apollo/server/dto/deezer/ChartD.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+import java.util.List;
+
+public class ChartD {
+    @JsonDeserialize(using = TracksDeserializer.class)
+    private List<TrackD> tracks;
+    @JsonDeserialize(using = PlaylistDeserializer.class)
+    private List<PlaylistD> playlists;
+//    private List<ArtistD> playlists;
+//    private List<AlbumD> playlists;
```
Comment (uaArsen) [inline server/src/main/java/org/room/apollo/server/dto/deezer/ChartD.java:15]: maybe remove this commented code, and from other classes as well.

Comment (AlexeyZatsepin) [reply server/src/main/java/org/room/apollo/server/dto/deezer/ChartD.java:15]: i will use it in future

Comment (uaArsen) [reply server/src/main/java/org/room/apollo/server/dto/deezer/ChartD.java:15]: ok...

**Review by uaArsen — APPROVED** [review]

**Pull request merged**